examples/onnx-distilbert-vanilla/src/main.rs

```diff
@@ -0,0 +1,26 @@
+use tract_onnx::prelude::*;
+
+fn main() -> TractResult<()> {
+    let model = tract_onnx::onnx()
+    // load the model
+    .model_for_path("bert_for_classification.onnx")?    // Model : DistilBERT; HuggingFace transformers
```

Comment:
Please add a minimum README file to the example to tell how to download the model.